s3repository/s3repository.go

```diff
@@ -0,0 +1,159 @@
+package s3metadatarepository
```

Comment:
Isn't the package supposed to match the folder name?

Reply:
yeah i renamed it, good catch